### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by simonepri**
## Changes

- Add `bugs` property
- Use compact syntax for contributors
- Add `keyboard` property
- Add `main` and `bin` property
- Add `files` property removing `.npmignore`
- Use shorthand for `repository` property
- Add myself to the contributors array

Ref #26 

Please merge this before #27 

cc @nelsonic @indexzero @zixia @soyuka

### CODE REVIEW

**Comment by simonepri** [comment]
Test fails because they are non deterministic, they are not related with the changes done in this PR.

**Comment by simonepri** [comment]
cc @nelsonic @indexzero @zixia @soyuka

**Comment by simonepri** [comment]
@nelsonic Can I add you as contributor?

@indexzero Any chances to add one of @zixia or @nelsonic or @simonepri inside the contributors on `npm`?

File: package.json [hunk 1]
```diff
@@ -2,32 +2,45 @@
...
+    "node": ">= 0.10"
   },
   "scripts": {
     "_comment": "https://github.com/gotwarlost/istanbul#usage-on-windows",
     "test": "istanbul cover node_modules/tape/bin/tape test/test.js test/direct.js",
     "coverage": "npm test && istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
     "codeclimate": "cross-env CODECLIMATE_REPO_TOKEN=84436b4f13c70ace9c62e7f04928bf23c234eb212c0232d39d7fb1535beb2da5 codeclimate < coverage/lcov.info"
   },
+  "dependencies": {
+    "event-stream": "~3.3.0"
```
Comment (nelsonic) [inline package.json]: It's _unclear_ from `this` PR where `event-stream` is being used. 
Perhaps consider adding detail to the PR description as to why/where it is required. 🤔

Comment (simonepri) [reply package.json]: event-stream is already in the package.json
I just reordered the keys

**Pull request merged**